### PR TITLE
Feature/shopping cart completed order page

### PR DESCRIPTION
**Pull request by DarinHajou**
Crated the page  where the order is completed and prompts the user to make a payment

### CODE REVIEW

**Comment by swoocn** [comment]
@DarinHajou - 

Accessible via **http://localhost:4200/shop/order-details**.

image

**Comment by swoocn** [comment]
I managed to link the shopping cart process, with the starting point accessible @ **http://localhost:4200/shop/shopping-cart**.  Please note that the data is currently static and are considered placeholders until we get the backend integrated to start working with 'real' data.

image

@zolcsi - Hope you have been well!  It's totally up to you, but if you see anything that could be refactored/ refined, then please feel free to adjust.  Thx!

**Review by swoocn — APPROVED** [review]
Angular conversion is complete to my knowledge.  It looks good to me!  Let me know what you think otherwise you have my approval.

File: frontend/src/app/shop/shopping-cart/order-details/order-details.component.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+export class OrderDetailsComponent implements OnInit {
+  ngOnInit(): void {
+    this.setupOrderButton();
+  }
+
+  /**
+   * Sets up the event listener for the order button.
+   */
+  setupOrderButton() {
```
Comment (zolcsi) [inline frontend/src/app/shop/shopping-cart/order-details/order-details.component.ts:21]: just a comment: it's not really the angular way, but if it works, all good.
In the html you can put a (click)="setupOrderButton()" and this method will be atumagically called, without document queries. Not that this is wrong here, just the angular way is simpler a bit

Comment (swoocn) [reply frontend/src/app/shop/shopping-cart/order-details/order-details.component.ts:21]: I will consider this feedback in the next revision.  Thanks!

**Pull request merged**